images/retry-trigger-jobs/retry_trigger_jobs.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import json
+import os
+import urllib.parse
+from datetime import datetime, timedelta, timezone
+from requests import Session
+from requests.adapters import HTTPAdapter, Retry
+
+import sentry_sdk
+
+
+sentry_sdk.init(traces_sample_rate=1.0)
```

Comment:
Since this is running every 15 mins, it makes sense to lower the traces_sample_rate here to avoid spamming sentry with too many events.
```suggestion
sentry_sdk.init(traces_sample_rate=0.01)
```